packages/grafana-ui/src/components/Logs/LogLabels.tsx

```diff
@@ -1,41 +1,71 @@
 import React, { FunctionComponent } from 'react';
 import { css, cx } from 'emotion';
-import { Labels, LogRowModel } from '@grafana/data';
+import { Labels } from '@grafana/data';
 
-import { LogLabel } from './LogLabel';
 import { stylesFactory } from '../../themes';
+import { Themeable } from '../../types/theme';
+import { GrafanaTheme } from '@grafana/data';
+import { selectThemeVariant } from '../../themes/selectThemeVariant';
+import { withTheme } from '../../themes/ThemeContext';
 
-const getStyles = stylesFactory(() => ({
-  logsLabels: css`
-    display: flex;
-    flex-wrap: wrap;
-  `,
-}));
+const HIDDEN_LABELS = ['level', 'lvl', 'filename'];
 
-interface Props {
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    logsLabels: css`
+      display: flex;
+      flex-wrap: wrap;
+      font-size: ${theme.typography.size.xs};
+    `,
+    logsLabel: css`
+      label: logs-label;
+      display: flex;
+      padding: 0 2px;
+      background-color: ${selectThemeVariant({ light: theme.colors.gray5, dark: theme.colors.dark6 }, theme.type)};
+      border-radius: ${theme.border.radius};
+      margin: 0 4px 2px 0;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      overflow: hidden;
+    `,
+    logsLabelValue: css`
+      label: logs-label__value;
+      display: inline-block;
+      max-width: 20em;
+      text-overflow: ellipsis;
+      overflow: hidden;
```

Comment:
I would personally like, if `no unique labels` would have the same font sizing as log labels - so I would add here `font-size: ${theme.typography.size.xs};`

![image](https://user-images.githubusercontent.com/30407135/70621214-5c81ec00-1c19-11ea-8b24-c9ceadafc961.png)